com.tdameritrade.ThinkOrSwim.yaml

```diff
@@ -0,0 +1,57 @@
+---
+app-id: com.tdameritrade.ThinkOrSwim
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: thinkorswim
+separate-locales: false
+
+tags:
+  - proprietary
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --persist=.thinkorswim
+  - --socket=x11
```

Comment:
can you add --share=ipc as well with socket=x11 to access x11 acceleration. (see https://docs.flatpak.org/en/latest/sandbox-permissions.html for more)

Reply:
Added, thanks for the pointer \o/